code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -23,7 +23,11 @@
 		return
 	A.attack_ghost(src)
 	if(ishuman(A) && in_range(src, A))
-		Harvest(A)
+		if(isLivingSSD(A))
+			if(client && client.send_ssd_warning(A)) //Do NOT Harvest(SSD people)
+				return
+		else
```

Comment:
To summarize what Ryan noticed. The way the if statement, inner, if statement and this else is statement are set up, prevents you from actually harvesting an SSD person.

Reply:
Updated